CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v2.0.1 2024-02-22
+- Handle response code for empty strings and blank responses
```

Comment:
Where is code changes for this? 🤔

Reply:
Here #169 

I was not aware that feature PR were also Release PR's... So I didn't bumped the version and didn't add a changelog.